src/nats/mod.rs

```diff
@@ -0,0 +1,63 @@
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "nats";
+const TAG: &str = "2.10.14";
+
+/// Nats image for [testcontainers](https://crates.io/crates/testcontainers).
+///
+/// This image is based on the official [Nats](https://hub.docker.com/_/nats) image.
+#[derive(Debug, Default)]
+pub struct Nats;
```

Comment:
Would be nice to make it not exhaustive in order to prevent backwards incompatible changes.

This way we won't break the API if we decide to add fields. `Nats::default().start()` will work, while `Nats.start()`

E.g:

```suggestion
pub struct Nats {
  _priv: (),
}
```

Reply:
[67020e1](https://github.com/testcontainers/testcontainers-rs-modules-community/pull/123/commits/67020e115076857b34d08546beed6aca3736464a)